lib/qa/authorities/linked_data/rdf_helper.rb

```diff
@@ -0,0 +1,82 @@
+require 'rdf'
+
+# Encapsulates processing of RDF results returned by the linked data authority.  This is used exclussively by Qa::Authorities::LinkedData::GenericAuthority
+# @see Qa::Authorities::LinkedData::GenericAuthority
+module Qa::Authorities
+  module LinkedData
+    module RdfHelper
```

Comment:
How about not using this as a mixin? That would improve the encapsulation you're talking about above.  Can you make the methods module methods and public (e.g.: `RdfHelper.get_linked_data(url)`)? In this way we know that this module has no dependency on GenericAuthority.